lib/beaker/hypervisor/aws_sdk.rb

```diff
@@ -405,7 +400,12 @@ def launch_all_nodes
           instance = create_instance(host, ami_spec, nil)
           instances.push({:instance => instance, :host => host})
         end
+        # Add metadata tags to each instance
```

Comment:
oh, duh, I should have read your first comment. do you know if that restriction is lifted in the v2 of this gem?